cmd/webrtc-load-tester/roles/orchestrator.go

```diff
@@ -269,6 +273,17 @@ func playerJobSpec(args loadTestArguments, region string, viewers int, playbackI
 	numTasks := viewers / args.Playback.ViewersPerWorker
 	timeout := args.TestDuration + 10*time.Minute
 
+	playbackURL := ""
+	if args.Playback.ManifestURL != "" {
+		u, err := url.Parse(args.Playback.ManifestURL)
+		if err != nil {
+			log.Println("failed to parse URL ", args.Playback.ManifestURL, err)
+		} else {
+			u = u.JoinPath("video+" + playbackID)
```

Comment:
If it seems helpful, a more flexible alternative would be for the input `ManifestURL` to be an `fmt`-format based string where the playback ID gets injected. e.g. this case would be a configuration like `livepeer.com/hls/video+%s` which we build with `fmt.Sprintf()` here.

Reply:
Yeah that makes sense actually and makes the code simpler